linkerd/proxy/http/src/orig_proto.rs

```diff
@@ -107,6 +107,7 @@ where
                 } else if orig_proto == "HTTP/1.0" {
                     Some(http::Version::HTTP_10)
                 } else {
+                    debug!(%orig_proto, "Malformed header value");
```

Comment:
IMO this should probably be
```suggestion
                    warn!(%orig_proto, "Malformed header value");
```
The reason I suggested we continue to emit a separate event this case is because the event won't be seen with the default filter configuration.

Reply:
What information does this provide a user? What if we change the values a server may emit and an older client talks to a newer server? I think it's useful to be able to know this when debugging; but I don't think this adds any value to users.

Reply:
Okay, that's fair. I was mostly just thinking that if we thought this should be a warning at some point in the past, it should continue to be one, but your rationale makes sense.